AppTaskQueue/appscale/taskqueue/distributed_tq.py

```diff
@@ -469,7 +470,7 @@ def __bulk_add(self, source_info, request, response):
       if (add_request.has_mode() and
           add_request.mode() == taskqueue_service_pb.TaskQueueMode.PULL):
         queue = self.get_queue(add_request.app_id(), add_request.queue_name())
-        if not isinstance(queue, PullQueue):
+        if not isinstance(queue, (PullQueue, PostgresPullQueue)):
```

Comment:
Perhaps not relevant for this pull request but there is a similar check in fetch_queue_stats that may also need updating.

Reply:
Thanks, I didn't notice that